packages/react-core/src/evm/hooks/wallet.ts

```diff
@@ -93,3 +95,33 @@ export function useAddress(): string | undefined {
 export function useChainId(): number | undefined {
   return useThirdwebConnectedWalletContext().chainId;
 }
+
+/**
+ * Hook for accessing the active Chain the current wallet is connected to
+ *
+ * ```javascript
+ * import { useActiveChain } from "@thirdweb-dev/react-core"
+ * ```
+ *
+ * @example
+ * You can get the chain of the connected wallet by using the hook as follows:
+ * ```javascript
+ * import { useActiveChain } from "@thirdweb-dev/react-core"
+ *
+ * const App = () => {
+ *   const chain = useActiveChain()
+ *
+ *   return <div>{chain.chainId}</div>
+ * }
+ * ```
+ * @public
+ */
+export function useActiveChain(): Chain | undefined {
+  const chainId = useThirdwebConnectedWalletContext().chainId;
```

Comment:
if we make the context hold the full chain object (if that's possible) we can save ourselves a find call here

Reply:
@jnsdls Yeah, it's on my todo list. we will do that